src/py/migraphx_py.cpp

```diff
@@ -63,26 +64,12 @@ py::buffer_info to_buffer_info(T& x)
         strides.begin(), strides.end(), strides.begin(), [&](auto i) { return i * s.type_size(); });
     py::buffer_info b;
     visit_type(s, [&](auto as) {
-        // migraphx use int8_t data to store bool type, we need to
-        // explicitly specify the data type as bool for python
-        if(s.type() == migraphx::shape::bool_type)
```

Comment:
Without the if branch, boolean type in python will not work....

Reply:
I am not sure how this got deleted. Let me put it back.